### ISSUE TITLE
*feature* (presentation) new themes

### ISSUE THREAD

**Issue by RomaricMourgues** [ONLYOFFICE/DocumentServer#246]
I noticed that Presentation themes are a bit old school,
can I change the themes manually ? How can I do this ?

I found this directory : DocumentServer/sdkjs/slide/themes, but I don't fully understand how it works and how to reload DocumentServer with new themes.

A lot of free of charge slides templates are available here https://www.slidescarnival.com for example

**Comment by ShockwaveNN** [comment]
Sorry, therer is no way to change themes without rebuilding application.

**Comment by RomaricMollard** [comment]
Ok, I will rebuild the application, what are the base64 strings in each theme.js (PPTY;v1;122731;[...]) ?

**Comment by jiriks74** [comment]
For DocumentServer this is now possible. [Here](https://helpcenter.onlyoffice.com/installation/docs-community-add-themes-linux.aspx) is a official guide. It's quite easy. Beware that some prezentation themes are a bit buggy (so far I figured that it should be files which names start with a number or have a number in their name) - the behavior of bugged themes is as follows: you choose the theme you want, but another theme, not the one you chose, is applied. The theme preview is also of the one that gets applied instead. So far I wasn't bothered to try and find a workaround. I just deleted the broken theme and rebuilt all themes again and everything was fine.
